style.css

```diff
@@ -0,0 +1,425 @@
+body {
+  background: #e5e5e5;
+}
+
+.hex {
+  color: #6070ff;
+}
+
+a:hover {
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.main-wrapper {
+  background: white;
+  display: flex;
+  flex-direction: column;
+  border-bottom-left-radius: 80px;
+}
+
+.toolbar {
+  flex: 1 1 auto;
+  display: flex;
+  height: 48px;
+  justify-content: space-between;
+  align-items: center;
+  padding-bottom: 40px;
+}
+
+.logo-place {
+  margin-left: 12px;
+}
+
+.my-logo {
+  padding: 10px 12px 10px 12px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  text-decoration: none;
+}
+
+.headline {
+  width: 100%;
+  background-image: url("images/Header\ bg.png");
+  background-size: 380px 550px;
+  border-radius: 0 0 0 100px;
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  padding: 20px 10px 0 10px;
+  height: calc(718px - 60px);
+  background-repeat: no-repeat;
+  overflow: hidden;
+}
+
```

Comment:
- [OPTIONAL ] Nice design but we can make it more attractive and professional so that recruiters can reach you, kindly try to fix your background-image to match the Figma design :wink: 

 Your Design 
![Screenshot from 2023-02-24 10-32-52](https://user-images.githubusercontent.com/88894525/221140187-fede9350-1a99-42db-8c9c-8239cea5d485.png)

Figma Design
![Screenshot from 2023-02-24 10-34-37](https://user-images.githubusercontent.com/88894525/221140361-91e988a2-00a2-42fa-8e0a-64fb280fb425.png)